safe-global/safe-wallet-web
# fix: remove email option from recovery

## What it solves

Removal of email integration

## How this PR fixes it

This removes the email section from the email upsertiong flow, as well as in the settings of recovery and notifications.

There remains the following open tasks that need to first be discussed:

- Implementation of a blocking element in the upsertiong flow, e.g. "I accept the risks associated with recovery"
- Adjustment of the default delay values

## How to test it

- Observe no mention of emails, as well as high delays values in the recovery upsertion flow
- Observe no mention of emails in the recovery/notification settings areas

## Screenshots

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/22d21535-85d1-4672-a34c-d6104798f8ef)

## Checklist
